packages/esm-patient-clinical-view-app/src/family-history/family-history.component.tsx

```diff
@@ -0,0 +1,164 @@
+import { EmptyDataIllustration, EmptyState, ErrorState } from '@openmrs/esm-patient-common-lib';
+import {
+  InlineLoading,
+  Layer,
+  Tile,
+  Pagination,
+  DataTableSkeleton,
+  TableContainer,
+  Table,
+  DataTable,
+  TableHead,
+  TableRow,
+  TableHeader,
+  TableBody,
+  TableCell,
+} from '@carbon/react';
+import { useFamilyHistory } from '../hooks/useFamilyHistory';
+import PatientChronicDiseases from './patient-chronic-dieseas.component';
+import { FamilyHistoryConcepts } from '../types';
+
+interface FamilyHistoryProps {
+  patientUuid: string;
+  encounterTypeUuid: string;
+  formEntrySub: any;
+  launchPatientWorkspace: Function;
+}
+const FamilyHistory: React.FC<FamilyHistoryProps> = ({ patientUuid, encounterTypeUuid }) => {
+  const { t } = useTranslation();
+  const headerTitle = t('familyHistory', 'Family History');
+  const { relationShips, relationShipError, isLoadingRelationShips, isValidatingRelationShips } =
+    useFamilyHistory(patientUuid);
+  const [currentPage, setCurrentPage] = useState(1);
+  const [pageSize, setPageSize] = useState(10);
+  const headers = [
+    {
+      header: t('name', 'Name'),
+      key: 'name',
+    },
+    {
+      header: t('relation', 'Relation/Relative'),
+      key: 'relation',
+    },
+    {
+      header: t('age', 'Age'),
+      key: 'age',
+    },
+    {
+      header: t('alive', 'Alive'),
+      key: 'alive',
+    },
+    {
+      header: t('causeOfDeath', 'Specific Cause of Death'),
+      key: 'causeOfDeath',
+    },
+    {
+      header: t('chronicleDisease', 'Chronic Disease'),
+      key: 'chronicleDisease',
+    },
+  ];
+  const tableRows =
+    relationShips?.map((entry) => {
+      const patientUuid = entry.patientUuid;
+      return {
+        id: `${entry.uuid}`,
+        name: entry?.display?.split(' - ')[1] || entry?.display,
+        relation: entry?.relationshipType,
+        age: entry?.relativeAge ?? '--',
+        alive: entry?.dead ? t('dead', 'Dead') : t('alive', 'Alive'),
+        causeOfDeath: (
+          <PatientChronicDiseases patientUuid={patientUuid} conceptUuid={FamilyHistoryConcepts.reason_for_death} />
+        ),
+        //  causeOfDeath: <PatientChronicDiseases patientUuid={patientUuid} conceptUuid={FamilyHistoryConcepts.cause_of_death} />,
+        patientUuid: entry,
+        chronicleDisease: (
+          <PatientChronicDiseases patientUuid={patientUuid} conceptUuid={FamilyHistoryConcepts.chronic_concept} />
+        ),
+      };
+    }) ?? [];
+  if (isLoadingRelationShips) {
+    return (
+      <InlineLoading
+        status="active"
+        iconDescription={t('loading', 'Loading')}
+        description={t('loadingDescription', 'Loading data...')}
+      />
+    );
+  }
+  if (relationShipError) {
+    return <ErrorState headerTitle={t('errorFamilyHistory', headerTitle)} error={relationShipError} />;
+  }
+  if (relationShips.length === 0) {
+    return <EmptyState headerTitle={headerTitle} displayText={headerTitle} />;
```

Comment:
```suggestion
    return <EmptyState headerTitle={headerTitle} displayText="family history data" />;
```

which gives you:

![CleanShot 2024-01-19 at 12  45 32@2x](https://github.com/palladiumkenya/kenyaemr-esm-3.x/assets/8509731/5b5062f7-7619-4440-bb74-d4ef464838a3)